### DISCUSSION TITLE
Stackdriver Tracing datasource

### DISCUSSION THREAD

**Discussion by inigohu** [grafana/grafana#41821] [Data Sources]
**What would you like to be added**:

With the last Grafana version (v7.0.0), now is possible to visualize traces. At this time, only Jaeger and Zipkin datasources are supported. It would be great to add Stackdriver as Grafana Traces datasource also.

**Why is this needed**:

To centralize all observability dashboards in Grafana with metrics and traces (and logs https://github.com/grafana/grafana/issues/21370) using stackdriver as a backend.

**Comment by daniellee** [comment]
Hoping to do this later this year. Think that we will do Stackdriver Logs first though.

Ref: https://github.com/grafana/grafana/pull/23066

**Comment by vickyyyyyyy** [comment]
This is on our radar, we will update this discussion when we have any more information!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
